src/js/media-container.js

```diff
@@ -392,6 +396,90 @@ class MediaContainer extends window.HTMLElement {
     });
   }
 
+  handleKeyupEvent(e) {
+    if (this.contains(e.target) || this.shadowRoot.contains(e.target)) {
+      // ignore space bar and enter
+      if (e.target instanceof MediaChromeButton && [' ', 'Enter'].includes(e.key)) {
+        return;
+      } else if (e.target instanceof MediaChromeRange && ['ArrowLeft', 'ArrowRight', 'ArrowUp', 'ArrowDown'].includes(e.key)) {
+        return;
+      }
+
+      let eventName, currentTimeStr, currentTime, detail, evt;
+      const seekOffset = 10;
+
+      switch (e.key) {
+        case ' ':
+        case 'k':
+          eventName =
```

Comment:
all of these were copied from the corresponding buttons. It would be nice if we can share the code, but I'm not positive there's a great way to do it. We could do something like `PlayButton.prototype.handleClick.call(this, e)` or something. Thoughts?

Reply:
Glad you're thinking of that. For now I'd lean towards "duplicate code" for this PR and KISS functional-like approaches (some module that "owns" a bucket of functions that take an element and return an appropriate `CustomEvent` (imo the "`dispatchEvent`" side effect should be externalized, but I could be convinced otherwise)